spec/models/project_spec.rb

```diff
@@ -118,4 +118,18 @@
       expect(Comment.count).to be 0
     end
   end
+
+  describe 'similar_projects' do
+    before do
+      @project = create(:project)
```

Comment:
Instead of `@project` use `let(:project) { create(:project })` outside of `before` block.